server/merge_controller.go

```diff
@@ -0,0 +1,49 @@
+package server
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"gopkg.in/mgo.v2"
+)
+
+// MergeController manages the resource handles for a Merge.
+type MergeController struct {
+	session  *mgo.Session
+	fhirHost string
+}
+
+// NewMergeController returns a pointer to a newly initialized MergeController.
+func NewMergeController(session *mgo.Session, fhirHost string) *MergeController {
+	return &MergeController{
+		session:  session,
+		fhirHost: fhirHost,
+	}
+}
+
+// Merges 2 FHIR bundles of patient resources given the URLs to both bundles.
+func (m *MergeController) merge(c *gin.Context) {
+	source1 := c.Query("source1")
+	source2 := c.Query("source2")
+	c.String(http.StatusOK, "Merging records %s and %s", source1, source2)
```

Comment:
I'm going to assume that the fact that this doesn't actually do any merging is intentional.  And the same for the rest of the functions in this file.